_conferences/wordcamp-italia-2022.md

```diff
@@ -0,0 +1,13 @@
+---
+name: "WordCamp Italia 2022"
+website: https://italia.wordcamp.org/2022/
+location: Milano, Italia
+online: false
+
+date_start: 2022-11-11
+date_end:   2022-11-12
+
+cfp_start: 2022-09-01
+cfp_end:   2022-10-10
+cfp_site:  https://italia.wordcamp.org/2022/
```

Comment:
I wasn't able to find the CFP here. Seems like the CFP is already closed? If so, can we remove this link?